StoneMasonKarel.java

```diff
@@ -13,5 +13,66 @@
 public class StoneMasonKarel extends SuperKarel {
 
 	// You fill in this part
+	public void run(){
+	turnLeft();
+	threeTimesNewBeeper();
+	move();
+	turnRight();
+	fourTimesMove();
+	turnRight();
+	newBeeper();
+	move();
+	newBeeper();
+	move();
+	turnLeft();
+	fourTimesMove();
+	turnLeft();
+	twoTimesNewBeeper();
+	move();
+	newBeeper();
+	turnRight();
+	fourTimesMove();
+	turnRight();
+	move();
+	newBeeper();
+	move();
+	newBeeper();
+	turnLeft();
+	
+	}
+	
 
+	private void newBeeper() {
+		// TODO Auto-generated method stub
```

Comment:
What is this TODO?

TODO is used to write something later? Do you need to write some code later?